app/Http/Controllers/Tquery/AdminUserTqueryController.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace App\Http\Controllers\Tquery;
+
+use App\Http\Controllers\ApiController;
+use App\Http\Permissions\Permission;
+use App\Http\Permissions\PermissionDescribe;
+use App\Services\Tquery\OpenApiGet;
+use Illuminate\Http\JsonResponse;
+use stdClass;
+
+class AdminUserTqueryController extends ApiController
+{
+    protected function initPermissions(): void
+    {
+        $this->permissionOneOf(Permission::globalAdmin);
+    }
+
+    #[OpenApiGet(
+        path: '/api/v1/admin/user/tquery',
+        permissions: new PermissionDescribe(Permission::globalAdmin),
+        summary: 'All users tquery',
+        tag: 'Admin',
+    )]
+    public function get(): JsonResponse
+    {
+        return new JsonResponse([
+            'data' => [
+                'columns' => [
+                    ['name' => 'id', 'type' => 'uuid'],
+                    ['name' => 'name', 'type' => 'string'],
+                    ['name' => 'email', 'type' => 'string'],
+                    ['name' => 'lastLoginFacilityId', 'type' => 'string'], //todo: 'type' => 'facility'
```

Comment:
Nie wiem czy ten typ uuid ma w ogóle sens ale jeżeli ma to to jest uuid. A być może TODO to powinno być na przykład `type: uuid, entity: facility` czy coś takiego, żeby nie wsadzać teraz dowolnych stringów do type.

To samo niżej dla user.